core/src/main/java/org/testcontainers/dockerclient/UnixSocketClientProviderStrategy.java

```diff
@@ -6,31 +6,24 @@
 import org.apache.commons.lang.SystemUtils;
 import org.jetbrains.annotations.NotNull;
 
-import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
 @Slf4j
 public class UnixSocketClientProviderStrategy extends DockerClientProviderStrategy {
```

Comment:
I don't know; the proxied approach seems to be pretty solid and AFAICT the main benefit of the netty KQueue enhancement is design elegance - it lets us remove a hack (albeit a reliable one). 

If we can't remove the hack anyway, then do we gain much from having a new potential way of working on OS X?

I need to think about this when I don't have such a headache 😄